content/applications/hr/payroll.rst

```diff
@@ -457,22 +629,20 @@ advantage listed for a particular structure type is only available for that spec
    have meal vouchers. The opposite is true for someone hired under the :guilabel:`Intern` structure
    type. They would have meal vouchers available to them, not the use of a company car.
 
-To make a new advantage, click the :guilabel:`New` button, and enter the information in the fields.
-The required fields for an advantage are:
+To make a new benefit, click the :guilabel:`New` button, and enter the information in the fields.
+The required fields for a benefit are:
 
-- :guilabel:`Name`: Enter the name for the advantage.
-- :guilabel:`Advantage Field`: Select from the drop-down menu what field in the payslip this
-  advantage appears under.
-- :guilabel:`Advantage Type`: Select from the drop-down menu what type of advantage the benefit is.
-  Select from :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Advantages in Net`,
-  :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
-- :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
-  advantage applies to.
-- :guilabel:`Display Type`: Select from the drop-down menu how this advantage is displayed.
+- :guilabel:`Name`: enter the name for the benefit.
```

Comment:
I followed the same logic for the rules, which was to explain the required fields only instead of all of it.  I am updating it so it matches the rules method of explaining everything. Also, for the image, I took an image of a pre-configured benefit in the runbot that made sense. I chose the internet subscription since it was small/easy to see, since big screen grabs are hard to take. But I understand your points- and addressed them. Thanks!